addons/mail/static/src/core/web/store_service_patch.js

```diff
@@ -30,10 +35,26 @@ const StorePatch = {
         this.starred = Record.one("Thread");
         this.history = Record.one("Thread");
     },
-    async initialize() {
-        this.fetchStoreData("failures");
-        this.fetchStoreData("systray_get_activities");
-        await super.initialize(...arguments);
+    /**
+     * Override to initialise using the lazy_session to initialise the webclient part
+     * @return {Promise<void>}
+     */
+    async _fetchStoreDataDebounced() {
+        if (!this.fetchParams.includes("init_messaging")) {
+            return super._fetchStoreDataDebounced();
+        }
+        this.env.services["lazy_session"].getValue("store_data", (storeData) => {
+            this.insert(storeData);
+            // if the "init_messaging" isn't the only param
+            this.fetchParams = this.fetchParams.filter((param) => param !== "init_messaging");
+            if (this.fetchParams.length > 0) {
+                return super._fetchStoreDataDebounced();
+            }
+            this.fetchDeferred = new Deferred();
+            this.fetchParams = [];
+            this.fetchReadonly = true;
+            this.fetchSilent = true;
```

Comment:
Can be put in a function to minimize copy-paste with original code, e.g. `resetFetchState()`